src/v0/destinations/google_adwords_remarketing_lists/networkHandler.js

```diff
@@ -1,7 +1,8 @@
 const { httpSend, prepareProxyRequest } = require('../../../adapters/network');
-const { isHttpStatusSuccess } = require('../../util/index');
-
-const { REFRESH_TOKEN } = require('../../../adapters/networkhandler/authConstants');
+const {
+  isHttpStatusSuccess,
```

Comment:
using this?

Reply:
If you mean `isHttpStatusSuccess`, it is getting used in `garl/networkHandler.js`